### PR TITLE
feat(wallet): implement new design

### PR DESCRIPTION
**Pull request by mhuggins**
## Description
This implements a large chunk of the new wallet design. There are still some outstanding pieces, and some polish is needed (particularly around the donut chart), but I would prefer to tackle them as part of a follow-up PR.

This also fixes some little issues I came across in the process, such as not checking the number of decimal places for NEP5 tokens in the send panel, instead only checking if it's NEO/GAS.

## Motivation and Context
We're implementing the new designs into the client.

## How Has This Been Tested?
By using the client and writing some simple tests.

## Screenshots (if appropriate)
screen shot 2018-07-24 at 12 30 26 am

## Types of changes
- [ ] Chore (tests, refactors, and fixes)
- [x] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [x] Tests for the changes have been added (for bug fixes/features)

#### Documentation
- [ ] Docs need to be added/updated (for bug fixes/features)

## Closing issues
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "electron-is-dev": "0.3.0",
     "electron-json-storage": "4.1.1",
     "es6-promisify": "6.0.0",
+    "format-currency": "^1.0.0",
```
Comment (DalderupMaurice) [inline package.json]: pin deps :P

File: src/renderer/account/actions/pricesActions.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// @flow
```
Comment (DalderupMaurice) [inline src/renderer/account/actions/pricesActions.js]: Remove flow tag

File: src/renderer/account/actions/pricesActions.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  if (data.error) {
+    throw new Error(data.error);
+  }
+
+  return mapPrices(data, currency);
+}
+
+export const ID = 'prices';
+
+export default createActions(ID, ({ currency = 'usd' }) => () => getPrices(currency));
```
Comment (DalderupMaurice) [inline src/renderer/account/actions/pricesActions.js]: Is this something which should be determined by the currency settings later on?

File: src/renderer/root/stylesheets/global.scss [hunk 4]
```diff
@@ -1,7 +1,9 @@
-@import url('https://fonts.googleapis.com/css?family=Rubik:300,400,700');
+@import url('https://rsms.me/inter/inter-ui.css');
+// @import './fonts/inter-ui';
```
Comment (DalderupMaurice) [inline src/renderer/root/stylesheets/global.scss]: remove comment

**Comment by DalderupMaurice** [comment]
Looks good so far, although I've "ignore" the code regarding the donut thing.
There are also some pieces of code in comment (in scss especially)

I've ran it on my Mac with an empty account and it looks good so far (besides for the donut not showing because I don't have a balance.

1 Thing maybe worth mentioning is the proportions.
Looking at this screenshot below - the proportions/sizes look a little weird/empty on fullscreen.

image

File: src/renderer/account/actions/pricesActions.js (hunk 3 above)
Comment (mhuggins) [reply src/renderer/account/actions/pricesActions.js]: Yep, that's the idea!

**Comment by mhuggins** [comment]
@DalderupMaurice Thanks for taking a look and for the feedback!  I addressed the few issues you pointed out.  Also, to answer your specific questions in your last comment...

> I've ran it on my Mac with an empty account and it looks good so far (besides for the donut not showing because I don't have a balance.

Yeah, I reached out on figma about getting a zero-state, so we'll just have to accept the lack of donut chart until that's ready.

> 1 Thing maybe worth mentioning is the proportions.
> Looking at this screenshot below - the proportions/sizes look a little weird/empty on fullscreen.

Agreed!  I think what we should do is add a max-width to the panel and leave space on the sides, similar to the new settings screen mockups.  I'll see what @fabric-8 thinks.

**Pull request merged**